test-configs.yaml

```diff
@@ -47,9 +47,9 @@ file_systems:
     type: debian
     ramdisk: 'buster-igt/20190823.0/{arch}/rootfs.cpio.gz'
 
-  debian_stretch-v4l2_ramdisk:
+  debian_buster-v4l2_ramdisk:
```

Comment:
Please don't update the URLs in this PR, they are for the production rootfs images which don't exist yet.

Reply:
alright